Hunting Queries/Microsoft 365 Defender/Exfiltration/unusual-volume-of-file-sharing.yaml

```diff
@@ -0,0 +1,59 @@
+id: bafc1446-1cc4-4f6d-ad76-1250b8c3b60c
+name: Unusual volume of file deletion by user.
+description: |
+  This query looks for users performing file deletion activities. Spikes in file deletion observed from risky sign-in sessions are flagged here.
+  This applies to SharePoint and OneDrive users.
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - CloudAppEvents
+  - AADSignInEventsBeta
+tactics:
+- Exfiltration
+query: |
+  let usefulExtn= pack_array('csv', 'doc', 'docm', 'docx', 'dot', 'dotx', 'eml', 'pdf', 'pot', 'potm', 'potx', 'ppam', 'pps', 'ppsm', 'ppsx', 'ppt', 'pptm', 'pptx',
+    'psd', 'pst', 'pub', 'ppk', 'rar', 'rtf', 'txt', 'vsd', 'vsdm', 'vsdx', 'vss', 'vssm', 'vst', 'vstm', 'vstx', 'xla', 'xlam', 'xll', 'xlm', 'xls', 'xlsm', 'xlsx', 'xlt',
+    'xltm', 'xltx', 'xps', 'zip', 'xsl');
+  let sharingPerms= pack_array("AnonymousLinkCreated", "SharingInvitationCreated", "SharingSet");
+  let relevantAppIds = pack_array(int(20892), int(15600)); // App Ids for SharePoint and OneDrive
```

Comment:
Could you add any public MS DOC/ Learn references where it mentioned AppId values belongs to the First party sharepoint/onedrive apps so that customers/community can later update and customize.. 
Probably this one https://learn.microsoft.com/en-us/troubleshoot/azure/active-directory/verify-first-party-apps-sign-in

Reply:
Added reference to cloud app identifiers.